wagtail/wagtailadmin/templates/wagtailadmin/base.html

```diff
@@ -37,3 +48,55 @@
         </div>
     </div>
 {% endblock %}
+
+{# Animation easter-egg: #}
+{% block extra_js %}
+    <script>
+        $(function() {
+          var sensitivity = 8; // the amount of times the user must stroke the wagtail to trigger the animation
+
+          var $logoContainer = $('.wagtail-logo-container__desktop');
```

Comment:
Gratuitous jquery! Do you really need to bake in a dependency here? Would this work instead -->
https://codepen.io/anon/pen/vJZXQN?editors=0010